package.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "root",
+    "name": "one-snyk-to-rule-them-all-and-in-the-darkness-bind-them",
```

This is the main `package.json` it doesn't have anything to do with snyk.
```suggestion
    "name": "one-design-system-to-rule-them-all-and-in-the-darkness-bind-them",
```